packages/twenty-server/src/engine/core-modules/workspace/services/workspace.service.ts

```diff
@@ -129,45 +131,87 @@ export class WorkspaceService extends TypeOrmQueryService<Workspace> {
     sender: User,
   ): Promise<SendInviteLink> {
     if (!workspace?.inviteHash) {
-      return { success: false };
+      return {
+        success: false,
+        errors: ['Workspace invite hash not found'],
+        result: [],
+      };
     }
 
     const frontBaseURL = this.environmentService.get('FRONT_BASE_URL');
     const inviteLink = `${frontBaseURL}/invite/${workspace.inviteHash}`;
 
-    for (const email of emails) {
-      const emailData = {
-        link: inviteLink,
-        workspace: { name: workspace.displayName, logo: workspace.logo },
-        sender: { email: sender.email, firstName: sender.firstName },
-        serverUrl: this.environmentService.get('SERVER_URL'),
-      };
-      const emailTemplate = SendInviteLinkEmail(emailData);
-      const html = render(emailTemplate, {
-        pretty: true,
-      });
-
-      const text = render(emailTemplate, {
-        plainText: true,
-      });
-
-      await this.emailService.send({
-        from: `${this.environmentService.get(
-          'EMAIL_FROM_NAME',
-        )} <${this.environmentService.get('EMAIL_FROM_ADDRESS')}>`,
-        to: email,
-        subject: 'Join your team on Twenty',
-        text,
-        html,
-      });
+    const workspaceInvitationsPr = await Promise.allSettled(
```

Comment:
Pr?

Reply:
It's to indicate that is a type  `PromiseSettledResult`  and not a simple list of invitations